source/connect/general-connection.txt

```diff
@@ -11,3 +11,80 @@ General Connection
    :backlinks: none
    :depth: 1
    :class: singlecol
+
+The :guilabel:`General` connection tab allows you to select a 
+:guilabel:`Connection String Scheme` for your |compass-short| deployment. 
+
+Procedure
+---------
+
+.. procedure::
+   :style: normal
+
+   .. step:: Click :guilabel:`Advanced Connection Options`.
+
+      .. figure:: /images/compass/advanced-connection-options.png
+         :figwidth: 690px
+         :alt: New Advanced Connection Options
+
+   .. step:: Click the :guilabel:`General` tab.
+
+      .. procedure::
+         :style: connected
+
+         .. step:: Select a Connection String Scheme.
+
+            You can select one of the following options:
+
+            .. list-table::
+               :header-rows: 1
+               :class: border-table
+
+               * - Connection String Scheme
+                 - Definition
+
+               * - mongodb
+                 - :manual:`Standard Connection String Format </reference/connection-string/>`. 
+                   The standard format of the MongoDB connection URI is used to 
+                   connect to a MongoDB deployment: standalone, replica set, or a sharded 
+                   cluster. 
+
+               * - mongodb+srv 
+                 - :manual:`DNS Seed List Connection Format 
+                   </reference/connection-string/#dns-seed-list-connection-format>`. 
+                   The ``+srv`` indicates to |compass| that the hostname that follows 
+                   corresponds to a DNS SRV record.
+
+         .. step:: Enter a Hostname.
+
+            In the :guilabel:`Host` field, enter the hostname of the server where the 
```

Comment:
The host input allows a user to specify multiple hosts if they're not connecting with an srv connection. Maybe we can include something like this to describe that:
```suggestion
            In the :guilabel:`Host` field, enter the hosts or hostname of the server where the 
```

Reply:
Done.